code/datums/wounds/_wounds.dm

```diff
@@ -79,6 +79,8 @@
 	var/datum/surgery/attached_surgery
 	/// if you're a lazy git and just throw them in cryo, the wound will go away after accumulating severity * 25 power
 	var/cryo_progress
+	/// wait, can cryo even heal it at all?
+	var/can_cryo_heal = FALSE
```

Comment:
I think this should be a flag on ``wound_flags`` instead, like ``ACCEPTS_CRYO``.